Tests/UnitTests/Scripts/tests.js

```diff
@@ -19,6 +19,23 @@ mocha.setup({ ui: "bdd", reporter: BabylonReporter });
 
 const expect = chai.expect;
 
+describe("AbortController / AbortSignal", function () {
+    const controller = new AbortController();
+
+    it("should have aborted flag set false on initialization", function () {
+        expect(controller.signal.aborted).to.equal(false);
+    })
+
+    it("should not throw while aborting and correctly trigger callback", function (done) {
+        // Expect aborted to be true after abort()
+        controller.signal.onabort = () => {
+            expect(controller.signal.aborted).to.equal(true);
+            done();
+        }
+        controller.abort();
+    })
```

Comment:
This is a bit strange to have state that crosses multiple test cases. Now that it's one polyfill, maybe it's better if this is one test case instead of two.

Reply:
I can see a case for either tbh, but make sense to keep things nicely encapsulated. Done.